Velaptor/OpenGL/GLWindow.cs

```diff
@@ -70,7 +70,7 @@ internal sealed class GLWindow : VelaptorIWindow
     /// <param name="nativeInputFactory">Creates a native input object.</param>
     /// <param name="glInvoker">Invokes OpenGL functions.</param>
     /// <param name="glfwInvoker">Invokes GLFW functions.</param>
-    /// <param name="systemMonitorService">Manages the systems monitors/screens.</param>
+    /// <param name="systemMonitorService">Provides information about the system monitors.</param>
```

Comment:
Should the word _**system**_ be plural?

So it is like this? 👉🏼 _**Provides information about the systems monitors.**_